frontend/packages/console-shared/src/hooks/useResizeObserver.ts

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+
+export const useResizeObserver = (
+  callback: ResizeObserverCallback,
+  targetElement: HTMLElement | null = document.querySelector('#app'),
```

Comment:
Do not execute a query selector in the default value otherwise it's executed each time this function is called and requires a default value.
```
const element = React.useMemo(() => targetElement ?? document.querySelector('#app'));
```

Also, why `#app` instead of the body?

Reply:
used body instead of `#app`